com.unity.formats.alembic/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 ### Fixed
 - Fixed a bug that was converting previous Alembic instance in prefabs into out-of-project references.
+- Fixed a bug that was causing Alembic files to be re-imported on every SRP project start.
```

Comment:
Suggestion:
`- Fixed a bug that was re-importing Alembic files on every SRP project start.`